### DISCUSSION TITLE
Error when SetAttributeFilter on layer

### DISCUSSION THREAD

**Discussion by dcasalino** [MaxRev-Dev/gdal.netcore#172] [Q&A]
Hi all, i'm trying to call method SetAttributeFilter on a Layer but i obtain this error D:\a\gdal.netcore\gdal.netcore\build-win\gdal-source\ogr\ogrsf_frmts\openfilegdb\filegdbtable.cpp.

The filter is                         layer.SetAttributeFilter("GN_CLASS = 2")

What I'm doing wrong ?

**Comment by MaxRev-Dev** [comment]
I have no idea as you posted a path instead of error message. Also, it would be nice to have a code+file sample ([MRE](https://stackoverflow.com/help/minimal-reproducible-example)). 
You can ask this question in GDAL repo as well.

**Reply by dcasalino** [reply]
Sorry you're true.

I'm opening a GeoDB database and reading layers.
On a specific layer i want to filter by a attribute field but i obtain this "applicationexception" error

![error](https://github.com/user-attachments/assets/e48426b1-18aa-4137-a626-63f6ae4291bb)

**Reply by MaxRev-Dev** [reply]
So as you have the line number [2568](https://github.com/OSGeo/gdal/blob/957c178ecba2f5ac04b3c642dec6b03f7ea4cd6a/ogr/ogrsf_frmts/openfilegdb/filegdbtable.cpp#L2568), you can try to locate it in GDAL repo and eventually figure out what's happening.
I'm not sure what it represents, better ask GDAL maintainers
